### PR TITLE
Karel the LED project

### PR DESCRIPTION
**Pull request by fisherds**
New fun project that I use with my students on the first day.  Seems to work well so I wanted to share it with others.

### CODE REVIEW

File: docs/projects/karel.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+Help Karel make LED art!
+
+### ~
+
+**Karel the LED** can't turn right, but he can make some great LED art!
+
+![](/static/mb/projects/karel/hi.png "Hi")
+
+The goal of this activity is to download the JavaScript code given below onto a board,
+then USE the program to introduce new students to the Micro:bit.
```
Comment (pelikhan) [inline docs/projects/karel.md]: use ``@boardname@`` instead of ``micro:bit``

File: docs/projects/karel.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            break;
+        }
+    }
+
+    pressedAB() {
+        this.isKarelActive = !this.isKarelActive;
+    }
+
+}
+```
```
Comment (pelikhan) [inline docs/projects/karel.md]: Could you add a "about the authors" section? See https://github.com/Microsoft/pxt-microbit/blob/master/docs/projects/magic-button-trick.md

File: docs/projects/karel.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+![](/static/mb/projects/karel/hi.png "Hi")
+
+The goal of this activity is to download the JavaScript code given below onto a board,
+then USE the program to introduce new students to the Micro:bit.
+Students will not do the coding this time.  They will be users to get familiar with the board.
+
+### How to play
+
+* ``A button`` **Turn Left**<br>
```
Comment (pelikhan) [inline docs/projects/karel.md]: do not use ``<br>``

File: docs/projects/karel.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+Does not draw anything just changes the direction Karel (the flashing led) is facing
+* ``B button`` **Move Forward**<br>
```
Comment (pelikhan) [inline docs/projects/karel.md]: do not use ``<br>``

File: docs/projects/karel.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+Moves Karel forward one step and leaves the LED on behind him
+* ``shake`` **Jump**<br>
```
Comment (pelikhan) [inline docs/projects/karel.md]: do not use ``<br>``

File: docs/projects/karel.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+Moves Karel forward one step without leaving the LED on behind him
+* ``A+B button`` **Hide Karel**<br>
```
Comment (pelikhan) [inline docs/projects/karel.md]: do not use ``<br>``

**Comment by fisherds** [comment]
Thanks for the feedback @pelikhan.  I added those changes, but just let me know if you think of anything else.  Happy to make edits.

**Pull request merged**